.github/workflows/build-image.yml

```diff
@@ -0,0 +1,29 @@
+name: Build Image
+
+on:
+  push:
+    tags:
+      - '*'
+  pull_request:
+    branches: [master]
+
+jobs:
+  publish-site:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Authenticate package registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login https://docker.pkg.github.com -u ${GITHUB_ACTOR} --password-stdin
+
+      - name: Set up ghost config
+        env:
+          GHOST_CONFIG: ${{ secrets.GHOST_CONFIG }}
+        run: echo "$GHOST_CONFIG" > ghostConfig.json
+
+      - name: Build dist
+        run: docker build -t docker.pkg.github.com/cuhacking/main-website/main-website .
+
+      - name: Push image
+        if: success()
+        run: docker push docker.pkg.github.com/cuhacking/main-website/main-website:latest
```

Comment:
Are images in the registry displayed anywhere? As in, is there a way to verify that the image is in the registry on our own (before pulling from the openstack machine)?

Reply:
For the repo: https://github.com/cuhacking/main-website/packages/

For the whole org: https://github.com/orgs/cuhacking/packages

Reply:
It's a normal docker registry, so I believe it supports all the same API endpoints as any other registry.